src/DataManagement/Entries/EntryList.cs

```diff
@@ -79,5 +82,28 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        [SuppressMessage("ReSharper", "PossibleNullReferenceException")]
+        public static IEntryList RandomList(IConfiguration configuration, EntryType entryType)
+        {
+            IEntryList list = new EntryList(entryType);
+            var section = configuration.GetSection(entryType.ToString());
```

Comment:
what is this line ? :D

Reply:
The configuration object is used to access the appsettings file, from which we take the section of either 'Income' or 'Expense' and use the values in it to set the names of the entries in the list